### PR TITLE
removing actionbar sherlock from the project and replacing it with appcompat

### PR DESCRIPTION
**Pull request by reidbaker**
DO NOT PULL THIS IN.
I am putting it up so that yall can see what the first round of changes looks like.

replace abs with appcompat, compiles, doesnt run because search is missing styling, docs not updated

### CODE REVIEW

**Comment by dgreenhalgh** [comment]
Thanks for putting this up, @reidbaker

**Comment by dgreenhalgh** [comment]
What's the status of this work, @reidbaker?

File: SoundStream/res/values/colors.xml [hunk 1]
```diff
@@ -33,6 +33,11 @@
     <color name="playbar">#DEDEDE</color>
     <color name="accent_blue">#33008AA6</color>
     <color name="medium_gray">#EEEEEE</color>
-    <color name="link_blue">#5858F8</color>
+    <color name="link_blue">#5858F8</color>  
+    <!-- Refactor colors -->
```
Comment (theJenix) [inline SoundStream/res/values/colors.xml:37]: Is this so that we maintain the same app appearance after removing ABS?  If so, is there something more standard we can switch to in AppCompat, at the expense of changing the app a bit?

File: SoundStream/src/com/thelastcrusade/soundstream/CoreActivity.java [hunk 2]
```diff
@@ -195,7 +192,8 @@ public boolean onCreateOptionsMenu(Menu menu) {
     }
     
     private boolean isConnectActive(){
-        return getSupportFragmentManager().findFragmentByTag(Transitions.CURRENT_CONTENT) instanceof ConnectFragment;
+        return true;
```
Comment (theJenix) [inline SoundStream/src/com/thelastcrusade/soundstream/CoreActivity.java]: I know this code is still in development, but I wanted to tag this so we don't overlook it.  This needs to be corrected, or commented.

File: SoundStream/src/com/thelastcrusade/soundstream/components/PlaylistFragment.java [hunk 3]
```diff
@@ -253,7 +253,7 @@ else if (entry.isPlayed()) {
                 element.findViewById(R.id.now_playing).setVisibility(View.INVISIBLE);
             } 
             else {
-                element.setBackgroundColor(getResources().getColor(com.actionbarsherlock.R.color.abs__background_holo_light));
+//                element.setBackgroundColor(getResources().getColor(com.actionbarsherlock.R.color.abs__background_holo_light));
```
Comment (theJenix) [inline SoundStream/src/com/thelastcrusade/soundstream/components/PlaylistFragment.java]: TODO: needs fixing or removal/comment.

**Comment by theJenix** [comment]
All in all this looks like a pretty straightforward refactor (provided the code behind the scenes works about the same).  There are a few TODOs, and it looks like some styles or style methods aren't defined in AppCompat, but given that AppCompat is Google's supported library for the action bar, my guess is this will be pretty easy to validate once it's done.

File: SoundStream/res/values/colors.xml (hunk 1 above)
Comment (reidbaker) [inline SoundStream/res/values/colors.xml:37]: Probably but lets have that be another pull request

File: SoundStream/src/com/thelastcrusade/soundstream/CoreActivity.java (hunk 2 above)
Comment (reidbaker) [inline SoundStream/src/com/thelastcrusade/soundstream/CoreActivity.java]: This is why I shouldnt try an do a refactor in multiple sittings.

File: SoundStream/src/com/thelastcrusade/soundstream/components/PlaylistFragment.java (hunk 3 above)
Comment (reidbaker) [inline SoundStream/src/com/thelastcrusade/soundstream/components/PlaylistFragment.java]: should be fixed now
